app/misc/graphing.ts

```diff
@@ -257,6 +277,9 @@ function sortBasicGraph() {
         if (idList[i] in branchIds) {
             bsNodes.update({id: branchIds[idList[i]], y: (i + 0.7) * spacingY})
         }
+        if (idList[i] in tagIds) {
+            bsNodes.update({id: tagIds[idList[i]], y: (i + 0.7) * spacingY, x: (i + 0.7) * spacingX})
```

Comment:
A comment would be nice here.

Reply:
Unfortunately I'm not entirely sure what this if statement does. It has to exist for everything to work, but it's just copied from the branch behaviour.

Reply:
Sorry I was referring to to bsNodes.update(...spacingY, spacing X...) that I didn't quite understand.